test/wait_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
+)
+
+var (
+	// This is a "hack" to make the example "look" like tests
```

Comment:
> maybe add a bit more here, i think this is just to fool the coverage tool? we could explain that here?

Actually I wished :upside_down_face: but it doesn't seem to fool the coverage (as it fails.). Examples do not take `t *testing.T` in their signature so… I needed a way to make the example "look like test" in the godoc and still compile.

I need to add real test to `wait.go` anyway (because the coverage tooling sees it as a new file and fail because it has 0 coverage :sweat_smile:) :wink: